v2_jobs_test.go

```diff
@@ -36,6 +42,11 @@ func TestJobs_List(t *testing.T) {
 	})
 
 	assert.NilError(t, err)
+	ti, _ := time.Parse("2006-01-02T15:04:05", "2024-01-18T01:27:49")
+
+	var tdt TaskDateTime
+	tdt.Time = ti
```

Comment:
There is an alternative where you don't have to use time.Parse:

```suggestion

	ti := TaskDateTime{
		Time: time.Date(2024, time.January, 18, 1, 27, 49, 0, time.UTC),
	}
	var tdt TaskDateTime
	tdt.Time = ti
```